docs/user-guide.md

```diff
@@ -0,0 +1,225 @@
+# TiDB Operator User Guide
+
+This document serves as a generic user guide for TiDB Operator. Basic understanding of [Kubernetes](https://k8s.io) and [TiDB](https://github.com/pingcap/tidb) is required. For quick start, please reference one of the following tutorials:
+
+* [Local DinD tutorial](./local-dind-tutorial.md)
+* [Google GKE tutorial](./google-kubernetes-tutorial.md)
+* [AWS EKS tutorial](./aws-eks-tutorial.md)
+
+## Requirements
+
+Before deploying the TiDB Operator, make sure the following requirements are satisfied:
+
+* Kubernetes v1.10 or later
+* [Helm](https://helm.sh) v2.8.2 or later
+* [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+* [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) enabled (optional)
+
+> **Note:** Though TiDB Operator can use network volume to persist TiDB data, it is highly recommended to set up [local volume](https://kubernetes.io/docs/concepts/storage/volumes/#local) for better performance. Because TiDB already replicates data, network volume will add extra replicas which is redundant.
+
+### Helm
+
+You can follow Helm official [documentation](https://helm.sh) to install Helm in your Kubernetes cluster. The following instructions are listed here for quick reference:
+
+1. Install helm client
+
+    ```
+    $ curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
+    ```
+
+    Or if you use macOS, you can use homebrew to install Helm by `brew install kubernetes-helm`
+
+2. Install helm server
+
+    ```shell
+    $ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/tiller-rbac.yaml
+    $ helm init --service-account=tiller --upgrade
+    $ kubectl get po -n kube-system -l name=tiller # make sure tiller pod is running
+    ```
+
+### Local Persistent Volume
+
+Local disks are recommended to be formatted as ext4 filesystem.
+
+Mount local ssd disks of your Kubernetes nodes at subdirectory of /mnt/disks. For example if your data disk is `/dev/nvme0n1`, you can format and mount with the following commands:
+
+```shell
+$ sudo mkdir -p /mnt/disks/disk0
+$ sudo mkfs.ext4 /dev/nvme0n1
+$ sudo mount -t ext4 -o nodelalloc /dev/nvme0n1 /mnt/disks/disk0
+```
+
+To auto-mount disks when your operating system is booted, you should edit `/etc/fstab` to include these mounting info.
+
+After mounting all data disks on Kubernetes nodes, you can deploy [local-volume-provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/local-volume) to automatically provision the mounted disks as Local PersistentVolumes.
```

Comment:
The user must use local-volume-provisioner  v2.2.0 or greater for `RetainPolicy` support.

Reply:
The following command uses the manifests we provided to deploy local volume provisioner. So it's already >= v2.2.0